src/detect-bsize.c

```diff
@@ -266,6 +268,68 @@ static DetectBsizeData *DetectBsizeParse (const char *str)
     return bsz;
 }
 
+static inline bool DetectCheckValue(const DetectBsizeData *bsz, uint64_t value)
+{
+    bool possible = true;
+    switch (bsz->mode) {
+        case DETECT_BSIZE_GT:
+            possible = true;
+            break;
+
+        case DETECT_BSIZE_EQ:
+            possible = value <= bsz->lo;
+            break;
+
+        case DETECT_BSIZE_RA:
+        case DETECT_BSIZE_LT:
+            possible = value< bsz->lo;
+            break;
+    }
+    return possible;
+}
+
+static bool DetectBsizeCheckContent(const SigMatch *sm, const DetectBsizeData *bsz)
+{
+    uint16_t bytes_required;
```

Comment:
if none of the members of the list are `content` the error message will use this uninitialized variable.

Reply:
also: the error won't make sense at all in this case